src/openzeppelin/security/initializable.cairo

```diff
@@ -1,17 +1,19 @@
-#[contract]
+#[starknet::contract]
 mod Initializable {
+    #[storage]
     struct Storage {
         initialized: bool
     }
 
-    #[internal]
-    fn is_initialized() -> bool {
-        initialized::read()
-    }
+    #[generate_trait]
+    impl StorageImpl of StorageTrait {
```

Comment:
```suggestion
    impl InternalImpl of InternalTrait {
```

This trait is for internal methods accessing storage, but external methods also access storage, so I think the name could be confusing. We also may have internal methods that don't access the storage, and shouldn't be part of this trait, so maybe `InternalStorageAccessTrait` is more descriptive, but too large IMO (is used a lot in tests). Wdyt?